azafea/event_processors/endless/activation/tests/test_activation_parsing.py

```diff
@@ -20,6 +20,8 @@ def test_from_bytes():
         'vendor': 'the vendor',
         'product': 'product',
         'release': 'release',
+        'serial': 'serial',
+        'mac_hash': 'mac_hash',
```

Comment:
```suggestion
        'mac_hash': 694551690,
```

This is a more realistic test, being an int not a string.